<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра &quot;Жизнь&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="controlLayout">
        <item>
         <spacer name="controlVerticalSpacer0">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="liveCellColourInfo">
          <property name="styleSheet">
           <string notr="true">font: 22pt &quot;Trebuchet MS&quot;;</string>
          </property>
          <property name="text">
           <string>Черные клеткие - живые</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deadCellColourInfo">
          <property name="styleSheet">
           <string notr="true">font: 22pt &quot;Trebuchet MS&quot;;</string>
          </property>
          <property name="text">
           <string>Белые клетки - мертвые</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="clickInfo">
          <property name="styleSheet">
           <string notr="true">font: 22pt &quot;Trebuchet MS&quot;;</string>
          </property>
          <property name="text">
           <string>Нажмите на клетку для изменения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="controlVerticalSpacer1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="rowCountSliderInfo">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Количество строк: 20</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="rowCountSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="columnCountSliderInfo">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Количество столбцов: 20</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="columnCountSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="cellPopulationOptionsLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="emptyGridButton">
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Очистить поле</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="randomGridButton">
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Случайно заполнить поле</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="controlVerticalSpacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startOrStopEvolvingButton">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>27</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Начать</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="controlVerticalSpacer3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="aboutButtons">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="aboutButton">
            <property name="text">
             <string>Об игре</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="authorButton">
            <property name="text">
             <string>Об авторе</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="controlVerticalSpacer4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="gridLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
